components/StyledButton.js

```diff
@@ -13,7 +14,9 @@ import StyledSpinner from './StyledSpinner';
  *
  * @see See [styled-system docs](https://github.com/jxnblk/styled-system/blob/master/docs/api.md) for usage of those props
  */
-const StyledButtonContent = styled.button`
+const StyledButtonContent = styled.button.attrs(({ ariaLabel }) => ({
+  'aria-label': ariaLabel,
+}))`
```

Comment:
Although the problem is solved, But I am not sure how to add them as Props, I tired all possible way out, but couldn't resolve.